CONTRIBUTING.md

```diff
@@ -10,6 +10,18 @@ of conduct can lead to a ban from the project and a report to GitHub.
 
 You can read the code of conduct [here](https://github.com/hikari-py/hikari/blob/master/CODE_OF_CONDUCT.md).
 
+# Versioning scheme
+
+This project follows the versioning scheme stated [here](https://semver.org/), but with a
+slight modification to the pre-release versioning: we denote all pre-releases with `.dev{num}`.
```

Comment:
Should probably link to the versioning format PEP this project follows since that's where ,dev comes from, it isn't a custom thing (might be PEP 440 but i'm not gonna check that rn).

Reply:
Yep, I initially had PEP 440 linked, but the current link explains simple versioning better as well as what is accepted in each release. Will add both just to be safe :)